extensions/sql-database-projects/src/sqldbproj.d.ts

```diff
@@ -179,6 +179,7 @@ declare module 'sqldbproj' {
 		sqlServer2017 = 'SQL Server 2017',
 		sqlServer2019 = 'SQL Server 2019',
 		sqlAzure = 'Microsoft Azure SQL Database',
-		sqlDW = 'Microsoft Azure SQL Data Warehouse'
+		sqlDW = 'Microsoft Azure SQL Data Warehouse',
+		sqlEdge = 'Microsoft Azure SQL Edge'
```

Comment:
just wanna make sure - "Microsoft Azure SQL Edge" doesn't show as an option in the Target Platform dropdown in the new project dialog or in the change target platform quickpick, right?

Reply:
https://user-images.githubusercontent.com/17187348/160208314-782f1140-111f-4533-8f9e-2142d7582892.png

Reply:
Btw, the official name for DW now is "Azure Synapse Dedicated SQL Pool"

https://docs.microsoft.com/en-us/azure/synapse-analytics/sql-data-warehouse/sql-data-warehouse-overview-what-is